form-editor-studio-plugin/src/main/joo/com/tallence/formeditor/studio/helper/FormElementsManager.as

```diff
@@ -145,5 +102,68 @@ public class FormElementsManager {
     return dragActiveVE;
   }
 
+  private function getRootNodeStruct():Struct {
+    if (formElements) {
+      return formElements.getValueAsStruct();
+    } else {
+      // if the sub struct is missing the sub struct has to be created
+      var root:Struct = (store.getRoot() as StructTreeNode).getValueAsStruct();
+      root.getType().addStructProperty(FormElementStructWrapper.FORM_ELEMENTS_PROPERTY);
+      return root.get(FormElementStructWrapper.FORM_ELEMENTS_PROPERTY);
+    }
+  }
+
+  /**
+   * Initializes the struct tree store. The store can be used to handle added or removed form elements, so that the
+   * value expression evaluating to the list of applied form elements can be updated. A functional value expression can
+   * not be used to evaluate the form elements directly, because the value expression would be evaluated too often.
+   * Every time a property of a form element is changed, for example the name of the field, the value expression would
+   * recalculate all form elements and the ui would be re-rendered.
+   */
+  private function initTreeStore():void {
+    var storeConfig:StructTreeStore = StructTreeStore({});
+    storeConfig.bindTo = contentVE;
+    storeConfig.propertyName = formDataStructPropertyName;
+    store = new StructTreeStore(storeConfig);
```

Comment:
der Store hält ja lediglich die einzelnen formElements in Form eines FormElementStructWrapper richtig?
Vielleicht könnte man die Variable auch umbenennen:
store -> formElementsWrapperStore

Reply:
erledigt.